unit-tests/src/test/java/org/eclipse/collections/impl/stream/PrimitiveStreamsTest.java

```diff
@@ -51,6 +51,7 @@
 import org.eclipse.collections.impl.factory.primitive.LongSets;
 import org.eclipse.collections.impl.factory.primitive.LongStacks;
 import org.eclipse.collections.impl.list.primitive.IntInterval;
+import org.eclipse.collections.impl.list.primitive.LongInterval;
```

Comment:
Please update the copyright to "Copyright (c) 2022 The Bank of New York Mellon and others."

Reply:
Done